src/main/java/com/demo/controller/DriverController.java

```diff
@@ -0,0 +1,69 @@
+package com.demo.controller;
+
+
+import com.demo.model.Driver;
+import com.demo.model.ParkingMeter;
+import com.demo.service.DriverService;
+import com.demo.service.ParkingEventService;
+import com.demo.service.ParkingMeterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class DriverController {
+
+    @Autowired
+    private DriverService driverService;
```

Comment:
jakoś inaczej nie da się wstrzyknąć?

Reply:
Nie byłem świadomy skutków wstrzykiwania poprzez pole. Poczytałem na ten temat i faktycznie, wstrzykiwanie przez pole nie jest najlepszym rozwiązaniem, gdyż można np. utworzyć obiekt będący w złym stanie (bez zainicjalizowanego pola) i dostać null pointer ex. Zostało to poprawione na wstrzykiwanie przez konstruktor.